catalog/pages/accordion/index.js

```diff
@@ -0,0 +1,35 @@
+import { pageLoader } from "catalog";
+import { ThemeProvider } from "styled-components";
+
+import { Button } from "../../../src/components/Button";
+import DisplayFor from "../../../src/components/DeviceSize"
+
+import Accordion from "../../../src/components/Accordion";
+import Chevron from "../../../src/blocks/EventAccordion/Chevron";
+import BottomSheet from "../../../src/blocks/EventAccordion/BottomSheet"
+import Event from "../../../src/blocks/EventAccordion/Event";
+import EventInfo from "../../../src/blocks/EventAccordion/EventInfo";
+import Text from "../../../src/components/Text";
+
+import { EXTRAS_DATA, EVENTS, EVENT_DATA, EVENTS_MANY } from "./mock";
+
+export default {
```

Comment:
What are you considering exporting from the library itself?

Reply:
I'm thinking EventListing.